src/structures/paging/page_table.rs

```diff
@@ -175,9 +191,38 @@ bitflags! {
         /// Can be only used when the no-execute page protection feature is enabled in the EFER
         /// register.
         const NO_EXECUTE =      1 << 63;
+
+        // Consider other as defined
+        #[cfg(feature = "dynamic_flags")]
+        const _ = !0;
```

Comment:
I don't think this is needed. We can use `PageTableFlags::from_bits_retain` which keeps all bits, including those not defined.

Reply:
I will check, I think what this does is that it makes from_bits_retain behave exactly as from_bits_truncate